jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -4360,7 +4360,7 @@ ConstantPool getConstantPool() {
 }
 /*[ENDIF] Sidecar19-SE*/
 
-/*[IF Valhalla-NestMates]*/
+/*[IF Java11]*/
```

Comment:
Is `pConfig VALHALLA` still supported assuming this change removes the natives affected from that configuration?
How about the check decorated by `Valhalla-NestMates` within `j.l.i.MethodHandles.checkAccess()`?

Reply:
Because we are not using the UMA spec files to turn on nestmates those flags will never be turned on. 

Ill have to go and fix the other nestmates flags in the jcl. Ill do that in a separate PR.

Reply:
fixed in this PR